packages/cli-test/src/cli/commands/platform.ts

```diff
@@ -203,6 +203,13 @@ export default {
     return new Promise((resolve, reject) => {
       // kill the shell process
       shell.kill(proc).then(() => {
+
+        // For the team being, we don't wait for the SLACK_TRACE_PLATFORM_RUN_STOP trace on Windows
+        // See HERMES-6136 for more information
```

Comment:
Let's not put internal JIRA references in our public tooling 👀

Reply:
Oh, shoot